client/QPKCS11.cpp

```diff
@@ -472,6 +473,11 @@ bool QPKCS11Stack::load(const QString &defaultDriver)
 		d->drivers.insert("/Library/mPolluxDigiSign/libcryptoki.dylib", "3B7B940000806212515646696E454944");
 	else
 		d->drivers.insert("/Library/OpenSC/lib/opensc-pkcs11.so", "3B7B940000806212515646696E454944");
+	d->drivers.insert("/Library/Frameworks/eToken.framework/Versions/Current/libeToken.dylib", "3BD5180081313A7D8073C8211030");
+	d->drivers.insert("/Library/Frameworks/eToken.framework/Versions/Current/libeToken.dylib", "3BD518008131FE7D8073C82110F4");
+	QVariantMap PKCS11 = Settings().value("PKCS11").toMap();
```

Comment:
Is this custom loading for OSX drivers documented anywhere? Shall there be a page in id.ee?

Reply:
Maybe document in wiki or support can create article also to id.ee